### PR TITLE
[ADD] partner_rank module (Manage partner's priority)

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: partner_rank/res_rank.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        'partner_ids': fields.one2many(
+            'res.partner', 'rank_id', 'Partner'),
+    }
+
+    _sql_constraints = [
+        ('priority_uniq', 'UNIQUE(priority)', 'The priority must be unique!'),
+    ]
+
+    def name_get(self, cr, uid, ids, context=None):
+        if context is None:
```
Comment (pedrobaeza) [inline partner_rank/res_rank.py:45]: You don't need to check context.

File: partner_rank/res_rank.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+#
+##############################################################################
+
+from openerp.osv import fields, orm
+
+
+class res_rank(orm.Model):
+    _description = 'Rank'
+    _name = 'res.rank'
+
```
Comment (pedrobaeza) [inline partner_rank/res_rank.py:28]: You can put order variable to set it based on priority.

**Comment by pedrobaeza** [comment]
This module can be interesting if it's connected through a CRM profiling module.

**Comment by hbrunn** [comment]
pylint says

```
partner_rank/tests/test_rank.py:34: [W0102(dangerous-default-value), Base_Test_rank.setUp] Dangerous default value {} as argument
```

**Comment by rafaelbn** [comment]
Hi @max3903 could you please review this PR in order to test it in runbot? Thanks!

**Comment by max3903** [comment]
Sure. Next week

Maxime Chambreuil

**Comment by max3903** [comment]
@rafaelbn It needs fixes more than reviews and I don't have write access to the branch any more. @joaoalf, @dufresnedavid up to you to fix it or close this PR...

**Comment by ghost** [comment]
@max3903 ok, I am closing this pr.

**Comment by ghost** [comment]
@joaoalf can you close the PR, I could delete the branch, but It seems I can not close the PR.

**Comment by rafaelbn** [comment]
OK,  I close.

**Pull request closed without merging**